lib/KListWithOverflow.vue

```diff
@@ -152,29 +161,32 @@
           itemsSizes.push(itemSize);
         }
 
+        if (this.overflowDirection === 'start') {
+              itemsSizes.reverse();
```

Comment:
We dont need to reverse the itemSizes because we are already reverting the `directionIndexes`, and when we iterate `directionIndexes`, we are using the same index to access the list.children which is not reverted and the itemSizes which is reverted. So we are not matching the same information for the children and its size.